### ISSUE TITLE
MongoDB views should be ignored introspecting the database

### ISSUE THREAD

**Issue by viniciusfdev** [prisma/prisma#16179]
### Bug description

I have a MongoDB database with a bunch of collections and some views, and when i try to make my first introspection i am getting the following error when running the **prisma db pull --force** command.

```md
Prisma schema loaded from src/main/config/prisma/schema.prisma
Environment variables loaded from .env
Datasource "db"

✖ Introspecting based on datasource defined in src/main/config/prisma/schema.prisma

Error: Error in connector: Error querying the database: Command failed (CommandNotSupportedOnView): Namespace tractian.Spider_Repositions is a view, not a collection)
```

I think the Prisma is not ignoring views, resulting in some error after acting on the view as if it were a collection.

### How to reproduce

1. Create a nodejs project with prisma included
2. Create a mongodb instance database (as a replica set)
3. Create collections, data and **views**
4. Run the command prisma db pull --force to generate the schema from the database

### Expected behavior

The error similiar to "Error in connector: Error querying the database: Command failed (CommandNotSupportedOnView): Namespace <view-name> is a view, not a collection)" should be logged. 

### Prisma information

```prisma
datasource db {
  provider = "mongodb"
  url      = env("MONGO_URL")
}

generator client {
  provider = "prisma-client-js"
}
```

### Environment & setup

- OS: ubuntu 22.04
- Database: MongoDB
- Node.js version: 14.17.6

### Prisma Version

```
4.5.0
```

**Comment by MusaliJagadeesh** [comment]
We are also facing same issue in our project.

**Comment by pimeys** [comment]
Confirmed.

**Comment by Jolg42** [comment]
Note: This is now fixed in `5.3.0-dev.9` if you want to try it out. Let us know if it works like expected.
It will be released in the official 5.3.0 version on Tuesday, September, 12th.
